core/src/main/java/org/ldaptive/asn1/DERParser.java

```diff
@@ -20,6 +20,12 @@
 public class DERParser
 {
 
+  /** Maximum supported path depth. */
+  private static final int MAX_PATH_DEPTH = 16;
+
+  /** Maximum number of tag permutations allowed. */
+  private static final int MAX_PERMUTATIONS = (int) Math.pow(2, MAX_PATH_DEPTH);
```

Comment:
I'm curious if this is strictly defensive.

Reply:
Yes, we don't see BER path depths that long in practice.